src/urban/restapi/services/content/address/base.py

```diff
@@ -41,6 +30,45 @@ def reply(self):
             )
         ]
 
+        return items
+        
+    def street_code(self):
+        street_code = self.request.get("street_code", None)
+        if not street_code:
+            return None
+
+        items = [
+            {"name": street.Title, "uid": street.UID}
+            for street in api.content.find(getStreetCode= [int(street_code)])
+        ]
+
+        return items
+
+
+    def reply(self):
+        """
+        End point to search street
+        callable with 'GET' and '@address'
+        Must add minimum one character as query parameters with the "term" key 
+        or the street code with the "street_code" key
+        You can provide a 'match' parameter to force the exact match on the result
+        You can provide a 'include_disable' parameter to expand search in disable streets
+
+        return an object with a list of object compose of uid and name of the street and
+            the count of item return
+        """
+
+        street_term = self.search_term()
+        street_code = self.street_code()
+
+        if not (street_term or street_code):
+            raise Exception('Must provide at least one query parameter with either the key "term" or "street_code"')
```

Comment:
This error message is not correct, if there is no result you will have an exception even if the parameters are provided.